explainaboard/processors/summarization.py

```diff
@@ -106,26 +106,26 @@ def default_features(cls) -> feature.Features:
                             setting=(),
                         ),
                     ),
-                    "oracle_score": feature.Value(
-                        dtype="float",
-                        description="the sample-level oracle score",
-                        is_bucket=True,
-                        bucket_info=feature.BucketInfo(
-                            method="bucket_attribute_specified_bucket_value",
-                            number=4,
-                            setting=(),
-                        ),
-                    ),
-                    "oracle_position": feature.Value(
-                        dtype="float",
-                        description="the sample-level oracle position",
-                        is_bucket=True,
-                        bucket_info=feature.BucketInfo(
-                            method="bucket_attribute_specified_bucket_value",
-                            number=4,
-                            setting=(),
-                        ),
-                    ),
+                    # "oracle_score": feature.Value(
```

Comment:
```suggestion
                    # TODO: these are commented out because the implementation is currently 
                    #              too slow to have them be enabled by default
                    # "oracle_score": feature.Value(
```